servers/dsub/jobs/controllers/aggregation_controller.py

```diff
@@ -94,22 +93,31 @@ def _to_aggregation(name, key, summary):
 
 def _to_top_labels_aggregations(label_summaries):
     # Rank the label summaries by the sum of valid item,
-    # where valid means a label has jobs more than _LABEL_MIN_COUNT_FOR_RANK.
+    # where valid means a label has jobs more than _min_count_for_rank().
     label_freq = {}
     for label, item in label_summaries.items():
         total_count = 0
-        for label_value, counts in item.items():
+        for _, counts in item.items():
             total_count += sum(
-                v for v in counts.values() if v > _LABEL_MIN_COUNT_FOR_RANK)
+                v for v in counts.values() if v > _min_count_for_rank())
         label_freq[label] = total_count
 
     aggregations = []
 
+    # Jobs with aggregation-testing-unique has higher priority
```

Comment:
I was suggesting your test would set something like:

```
app.app.config['AGGREGATION_FILTER_LABEL'] = "aggregation-testing-unique:<unique_test_value>"
```

Normally you would not set this config value, but if present, you pass it through to the job generator as a label filter. This option would likely never be used outside of testing, but is at least generic.

See test_token_label in the google test for a similar approach.

Reply:
Changed to 
```
app.app.config[
                'AGGREGATION_FILTER_LABEL'] = "aggregation-testing-unique:" + datetime.datetime.now().strftime('%Y%m%d_%H%M')
```